### DISCUSSION TITLE
CI/CD setup: NodeJS / Docker / Kubernetes

### DISCUSSION THREAD

**Discussion by TuriGuilano** [BretFisher/ama#134] [Q&A]
Dear Bret,

I am currently setting up my CI/CD process for my Node/Docker/Kubernetes stack. I read
a lot about CI/CD with Gitlab / Gitlab runners for Docker, what would you recommend?

Thanks for your time.

**Comment by BretFisher** [comment]
Any modern CI will do. GitLab is popular for sure. If you follow my bretfisher.com/node course you can do most CI things like tests, audits, and CVE scans inside the docker image build.

**Comment by TuriGuilano** [comment]
Cheers Bret, will check that one out for sure.
